tasks/main.yml

```diff
@@ -53,3 +53,14 @@
     - '{{ logrotate_applications }}'
   tags:
     - configuration
+
+- name: Set logrotate hourly state
```

Comment:
This is obsolete in my proposal.

Reply:
Fix in 17cc47173ce9945c74dd3629f6912df504a8eeeb